src/main/webapp/resources/js/components/Header/MainNavigation.jsx

```diff
@@ -1,227 +1,202 @@
 const isManager = isAdmin || window.TL._USER.systemRole === "ROLE_MANAGER";
 
 const textColor = theme === "dark" ? "#fff" : "#222";
 
-const MenuStyle = styled.div`
+const StyledHeader = styled(Header)`
+  position: fixed;
+  z-index: 1;
+  width: 100%;
   display: flex;
   justify-content: space-between;
-  justify-self: end;
-  align-items: center;
-  flex-wrap: wrap;
-  font-size: 2rem;
-  border-bottom: 5px solid ${primaryColour};
-  color: ${textColor};
 
-  .ant-menu-item,
-  .ant-menu-item-active,
-  .ant-menu-item-selected,
-  .ant-menu-item-only-child {
-    background-color: #001529 !important;
+  .ant-menu-item-active {
+    background-color: transparent !important;
   }
 
-  .global-search {
-    border-radius: 10px;
+  .anticon {
+    font-size: 20px;
+    color: ${grey6};
   }
 
-  .ant-dropdown-trigger {
-    color: ${textColor};
-    font-size: 1.5rem;
+  .ant-menu-submenu a {
+    color: ${grey6};
   }
-`;
 
-const MenuItemStyle = styled(Menu.Item)`
-  width: 50px !important;
-  padding: 0 !important;
+  .ant-menu-item:hover .anticon,
+  .ant-menu-submenu:hover .anticon {
+    color: ${grey1};
+  }
 `;
 
-const ProjectsMenu = (
-  <Menu key="projects" theme={theme}>
-    <Menu.Item key="project:list">
-      <a href={setBaseUrl("/projects")}>{i18n("nav.main.project-list")}</a>
-    </Menu.Item>
-    {isAdmin && (
-      <Menu.Item key="project:all">
-        <a href={setBaseUrl("/projects/all")}>
-          {i18n("nav.main.project-list-all")}
-        </a>
-      </Menu.Item>
-    )}
-    <Menu.Divider />
-    <Menu.Item key="project:sync">
-      <a href={setBaseUrl("/projects/synchronize")}>
-        {i18n("nav.main.project-sync")}
-      </a>
-    </Menu.Item>
-  </Menu>
-);
-
-const AnalysesMenu = (
-  <Menu theme={theme}>
-    <Menu.Item key="analyses:user">
-      <a href={setBaseUrl(`/analysis`)}>
-        {i18n("nav.main.analysis-admin-user")}
-      </a>
-    </Menu.Item>
-    {isAdmin && (
-      <Menu.Item key="analyses:all">
-        <a href={setBaseUrl("/analysis/all")}>
-          {i18n("nav.main.analysis-admin-all")}
-        </a>
-      </Menu.Item>
-    )}
-    <Menu.Divider />
-    <Menu.Item key="analyses:output">
-      <a href={setBaseUrl("/analysis/user/analysis-outputs")}>
-        {i18n("Analysis.outputFiles")}
-      </a>
-    </Menu.Item>
-  </Menu>
-);
-
-const UsersMenu = (
-  <Menu theme={theme}>
-    <Menu.Item key="user:users">
-      <a href={setBaseUrl("/users")}>{i18n("nav.main.users-list")}</a>
-    </Menu.Item>
-    <Menu.Item key="user:groups">
-      <a href={setBaseUrl("/groups")}>{i18n("nav.main.groups-list")}</a>
-    </Menu.Item>
-  </Menu>
-);
-
-const HelpMenu = (
-  <Menu theme={theme}>
-    <Menu.Item key="userguide">
-      <a
-        href="https://irida.corefacility.ca/documentation/user/user"
-        target="_blank"
-        rel="noreferrer"
+export function MainNavigation() {
+  return (
+    <StyledHeader>
+      <Menu
+        theme="dark"
```

Comment:
Change this `theme="dark"` to use the `theme` from `theme-utilities.js`

Reply:
Updated